### ISSUE TITLE
Crash on server with memoryleakfix

### ISSUE THREAD

**Issue by gvancamp** [SolidBlock-cn/BRRP#5]
[15:43:39] [main/INFO]: Loading Minecraft 1.19.3 with Fabric Loader 0.14.11
[15:43:39] [main/INFO]: Loading 42 mods:
	- better_runtime_resource_pack 0.8.2
	- fabric-api 0.69.1+1.19.3
	- fabric-api-base 0.4.19+8d1895cf85 via fabric-api
	- fabric-api-lookup-api-v1 1.6.18+49abcf7e85 via fabric-api
	- fabric-biome-api-v1 12.0.0+689f5e7185 via fabric-api
	- fabric-block-api-v1 1.0.3+12bfe4ea85 via fabric-api
	- fabric-command-api-v1 1.2.19+f71b366f85 via fabric-api
	- fabric-command-api-v2 2.1.15+49abcf7e85 via fabric-api
	- fabric-commands-v0 0.2.36+df3654b385 via fabric-api
	- fabric-containers-v0 0.1.44+df3654b385 via fabric-api
	- fabric-content-registries-v0 3.4.12+49abcf7e85 via fabric-api
	- fabric-convention-tags-v1 1.2.0+8e4e694f85 via fabric-api
	- fabric-crash-report-info-v1 0.2.12+aeb40ebe85 via fabric-api
	- fabric-data-generation-api-v1 11.1.1+112a38be85 via fabric-api
	- fabric-dimensions-v1 2.1.39+48349a3f85 via fabric-api
	- fabric-entity-events-v1 1.5.5+b83334a085 via fabric-api
	- fabric-events-interaction-v0 0.4.37+422b77fb85 via fabric-api
	- fabric-events-lifecycle-v0 0.2.39+df3654b385 via fabric-api
	- fabric-game-rule-api-v1 1.0.28+2894d6df85 via fabric-api
	- fabric-item-api-v1 2.1.5+4d2c606385 via fabric-api
	- fabric-item-group-api-v1 2.1.8+d3d735d085 via fabric-api
	- fabric-lifecycle-events-v1 2.2.8+2894d6df85 via fabric-api
	- fabric-loot-api-v2 1.1.18+75e9821185 via fabric-api
	- fabric-loot-tables-v1 1.1.22+9e7660c685 via fabric-api
	- fabric-message-api-v1 5.0.12+6ede1da985 via fabric-api
	- fabric-mining-level-api-v1 2.1.29+49abcf7e85 via fabric-api
	- fabric-networking-api-v1 1.2.14+d9ac3c3985 via fabric-api
	- fabric-networking-v0 0.3.31+df3654b385 via fabric-api
	- fabric-object-builder-api-v1 5.2.0+e9dee00485 via fabric-api
	- fabric-particles-v1 1.0.18+fbde993d85 via fabric-api
	- fabric-registry-sync-v0 2.0.1+2608564685 via fabric-api
	- fabric-rendering-data-attachment-v1 0.3.23+48349a3f85 via fabric-api
	- fabric-rendering-fluids-v1 3.0.16+3067f46385 via fabric-api
	- fabric-resource-conditions-api-v1 2.2.0+280be3ab85 via fabric-api
	- fabric-resource-loader-v0 0.10.3+a10d22bd85 via fabric-api
	- fabric-screen-handler-api-v1 1.3.10+689f5e7185 via fabric-api
	- fabric-transfer-api-v1 2.1.12+82a2701785 via fabric-api
	- fabric-transitive-access-wideners-v1 2.1.0+0efcd39b85 via fabric-api
	- fabricloader 0.14.11
	- java 17
	- memoryleakfix 1.19.3-0.7.0
	- minecraft 1.19.3
[15:43:39] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Games/Minecraft/test/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[15:43:39] [main/INFO]: Compatibility level set to JAVA_17
[15:43:40] [main/INFO]: BRRP data generation: PreLaunch
[15:43:43] [main/INFO]: Building unoptimized datafixer
[15:43:44] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[15:43:44] [main/INFO]: BRRP register - before vanilla
[15:43:44] [main/INFO]: BRRP register - after vanilla
[15:43:45] [main/INFO]: Loaded 7 recipes
[15:43:46] [main/INFO]: Loaded 1179 advancements
[15:43:46] [main/INFO]: Applied 0 biome modifications to 0 of 63 new biomes in 1.514 ms
[15:43:46] [Server thread/INFO]: Starting minecraft server version 1.19.3
[15:43:46] [Server thread/INFO]: Loading properties
[15:43:46] [Server thread/INFO]: Default game type: SURVIVAL
[15:43:46] [Server thread/INFO]: Generating keypair
[15:43:46] [Server thread/INFO]: Starting Minecraft server on *:25565
[15:43:46] [Server thread/INFO]: Using default channel type
[15:43:46] [Server thread/INFO]: Preparing level "world"
[15:43:47] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[15:43:48] [Worker-Main-16/INFO]: Preparing spawn area: 0%
[15:43:48] [Worker-Main-16/INFO]: Preparing spawn area: 0%
[15:43:48] [Worker-Main-20/INFO]: Preparing spawn area: 0%
[15:43:48] [Worker-Main-21/INFO]: Preparing spawn area: 12%
[15:43:49] [Server thread/INFO]: Time elapsed: 1892 ms
[15:43:49] [Server thread/INFO]: [MemoryLeakFix] Attempting to ForceLoad All Mixins and clear cache
[15:43:49] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.ExceptionInInitializerError: null
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:467) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.MixinServiceKnot.findClass(MixinServiceKnot.java:96) ~[fabric-loader-0.14.11.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.audit(MixinProcessor.java:254) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.audit(MixinTransformer.java:171) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.MixinEnvironment.audit(MixinEnvironment.java:1425) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at ca.fxco.memoryleakfix.MemoryLeakFix.forceLoadAllMixinsAndClearSpongePoweredCache(MemoryLeakFix.java:30) ~[memoryleakfix-1.19.3-0.7.0.jar:?]
	at net.minecraft.server.MinecraftServer.handler$zga000$onFinishedLoadingWorlds(MinecraftServer.java:4793) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3735(MinecraftServer.java:338) ~[server-intermediary.jar:?]
	at net.minecraft.class_3176.method_3823(class_3176.java:169) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:648) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:264) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalStateException: This registry can't create intrusive holders
	at net.minecraft.class_2370.method_40269(class_2370.java:374) ~[server-intermediary.jar:?]
	at net.minecraft.class_2248.<init>(class_2248.java:72) ~[server-intermediary.jar:?]
	at net.devtech.arrp.generator.BRRPCubeBlock.<init>(BRRPCubeBlock.java:29) ~[brrp-fabric-0.8.2-1.19.3.jar:?]
	at net.devtech.arrp.generator.BRRPCubeBlock.cubeAll(BRRPCubeBlock.java:42) ~[brrp-fabric-0.8.2-1.19.3.jar:?]
	at net.devtech.arrp.BRRPDevelopment.<clinit>(BRRPDevelopment.java:58) ~[brrp-fabric-0.8.2-1.19.3.jar:?]
	... 13 more
[15:43:49] [Server thread/ERROR]: This crash report has been saved to: C:\Games\Minecraft\test\.\crash-reports\crash-2022-12-15_15.43.49-server.txt
[15:43:49] [Server thread/INFO]: Stopping server
[15:43:49] [Server thread/INFO]: Saving players
[15:43:49] [Server thread/INFO]: Saving worlds
[15:43:50] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:overworld
[15:43:50] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_end
[15:43:50] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_nether
[15:43:50] [Server thread/INFO]: ThreadedAnvilChunkStorage (world): All chunks are saved
[15:43:50] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[15:43:50] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[15:43:50] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved

**Comment by TERABITDEFIANCE** [comment]
Im getting this same error. brrp 0.8.2. Fabric 0.14.12 server. memoryleakfix 0.7.0. Minecraft 1.19.2. Also many other mods but same error. World gets up to creation and suddenly saves and stops with the error above.

**Comment by SolidBlock-cn** [comment]
I noticed that there is following information:
```
at net.devtech.arrp.BRRPDevelopment.(BRRPDevelopment.java:58) ~[brrp-fabric-0.8.2-1.19.3.jar:?]
... 13 more
```
Could you please provide the content of the "13 more"? The class `BRRPDevelopment` _should not_ be loaded _outside of_ the development environment

**Comment by SolidBlock-cn** [comment]
I've test just now for BRRP 0.8.2, memory leak fix 1.19.3-0.8.0, and Fabric Loader 0.14.17. The game correctly started.
Please try updating Fabric Loader.

**Comment by SolidBlock-cn** [comment]
Duplicate with #3 .

Edit: That issue is 1.19.2, but the problem and cause are the same.
